simpa/core/device_digital_twins/digital_device_twin_base.py

```diff
@@ -229,10 +229,12 @@ def add_illumination_geometry(self, illumination_geometry, illuminator_position_
             msg = "The given illumination_geometry must not be None!"
             self.logger.critical(msg)
             raise ValueError(msg)
-        if np.linalg.norm(illumination_geometry.device_position_mm) == 0 and \
-                illuminator_position_relative_to_pa_device is not None:
-            illumination_geometry.device_position_mm = np.add(self.device_position_mm,
-                                                              illuminator_position_relative_to_pa_device)
+        if np.linalg.norm(illumination_geometry.device_position_mm) == 0:
```

Comment:
What if we explicitly want to set the device_position to [0, 0, 0] and it is different to self.device_position_mm? In this case, it would be overwritten, right?
Should we always initialise the illuminator/detector position with None? And if it is None, then we adjust?

Reply:
I think we can leave it as is for now if it boils down to a convention, but maybe we could open a low prior issue to follow up on this at some point?